src/SFA.DAS.AANHub.Application/Employers/Queries/GetEmployerMemberQueryValidator.cs

```diff
@@ -0,0 +1,22 @@
+﻿using FluentValidation;
+using SFA.DAS.AANHub.Domain.Interfaces.Repositories;
+
+namespace SFA.DAS.AANHub.Application.Employers.Queries
+{
+    public class GetEmployerMemberQueryValidator : AbstractValidator<GetEmployerMemberQuery>
+    {
+        public const string UserIdNotFoundMessage = "UserId was not found";
+        public GetEmployerMemberQueryValidator(IEmployersReadRepository employersReadRepository)
+        {
+            RuleFor(a => a.UserId)
+                .NotNull()
+                .NotEmpty()
+                .MustAsync(async (userId, cancellation) =>
```

Comment:
Where the userid is invalid, we should return a NOTFOUND response and hence this async check will have to be removed from here.

Reply:
Removed async check. Within  employercontrollertest.cs - renamed `Validates_UserId_NotNull_NotFound` to  `Validates_UserId_NotNull`